src/locales/en.json

```
@@ -333,10 +333,10 @@
     },
     "lengths": {
       "title": "Durations",
-      "shortest": "Shortest",
-      "short": "Short",
-      "medium": "Medium",
-      "long": "Long"
+      "shortest": "Shortest (< 30s)",
+      "short": "Short (30s - 2min)",
+      "medium": "Medium (2 - 10min)",
```

```suggestion
      "short": "Short (30s-2min)",
      "medium": "Medium (2-10min)",
```

I believe with these kinds of ranges there isn't generally a space.